assets/js/modules/analytics/components/dashboard/AnalyticsAdSenseDashboardWidgetTopPagesTable.js

```diff
@@ -141,16 +119,14 @@ class AnalyticsAdSenseDashboardWidgetTopPagesTable extends Component {
  * @return {(string|boolean|null)}  Returns a string with an error message if there is an error. Returns `false` when there is no data and no error message. Will return `null` when arguments are invalid.
  *                            string   data error message if it exists or unidentified error.
  *                            false    if no data and no error message
- *                            null     if invalid agument
+ *                            null     if invalid argument
  *
  */
 const getDataError = ( data ) => {
 	if ( data.code && data.message && data.data && data.data.status ) {
 		// Specifically looking for string "badRequest"
 		if ( 'badRequest' === data.data.reason ) {
-			return AnalyticsAdSenseDashboardWidgetTopPagesTable.renderLayout(
-				ctaWrapper( <AdSenseLinkCTA />, false, false, true )
-			);
+			return <AnalyticsAdSenseDashboardWidgetLayout>{ ctaWrapper( <AdSenseLinkCTA />, false, false, true ) }</AnalyticsAdSenseDashboardWidgetLayout>;
```

Comment:
Nit-picking, but let's wrap this into multiple lines for readability since it's now multiple elements.

Reply:
Updated.